spikingjelly/activation_based/model/__init__.py

```
@@ -0,0 +1,21 @@
+)
+
+__all__ = [
+    "SpikformerConv2dBN",
+    "SpikformerConv2dBNLIF",
+    "SpikformerPatchStem",
+    "SpikformerMLP",
+    "SpikformerBlock",
+    "Spikformer",
+    "spikformer_ti",
+    "spikformer_s",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)